thirdparty/cmdconfig/commands/cmdsource/cmdsource.go

```diff
@@ -29,6 +33,8 @@ func GetSourceRunner(name string) *SourceRunner {
 	}
 	c.Flags().StringVar(&r.FunctionConfig, "fn-config", "",
 		"path to function config file.")
+	c.Flags().BoolVar(&r.IncludeMetaResources,
+		"include-meta-resources", false, "include package meta resources in the command output")
```

Comment:
Why does value of flag help message differ from reference docs?

Reply:
`DIR` is required now as mentioned in https://github.com/GoogleContainerTools/kpt/pull/2032#discussion_r635658705.

https://github.com/GoogleContainerTools/kpt/issues/2159

Reply:
The help message is a brief description about the flag, meanwhile the reference doc includes details. This is the same for other commands.

Reply:
I don't agree with this. It's weird to see to a different flag description from (--help) and reference docs. @mortent @droot

Reply:
The current output of `--help` combines the reference doc and flag description. Isn't it verbose to have same sentences twice?

Reply:
Actually the real bug from an end user's perspective is that `flags` help gets printed twice in two different forms (detailed form picked from reference/*/.md and short form from the default commandline flags here). 

I don't know the real fix here, but @Shell32-Natsu can you file a bug for this, we need to address it   for all the commands.

Reply:
https://github.com/GoogleContainerTools/kpt/issues/2167